packages/cli/lib/errorHandlers/index.js

```diff
@@ -16,7 +16,10 @@ const i18nKey = 'lib.errorHandlers.index';
 function logError(error, context = {}) {
   debugError(error, context);
 
-  if (shouldSuppressError(error)) {
+  if (
+    shouldSuppressError(error, context) ||
+    shouldSuppressError(error, error.context)
```

Comment:
This was the cause `hs project upload` errors, the context just wasn't getting passed in here.  I added the second call here because the error may have it's own `context` prop in the even of the `HubSpotHttpError`